vpr/src/place/compressed_grid.h

```diff
@@ -32,6 +32,8 @@ struct t_compressed_block_grid {
     //stored sparsely, since we may not have full columns of blocks.
     //This makes it easy to check whether there exist
     std::vector<vtr::flat_map2<int, t_type_loc>> grid;
+
+    std::unordered_map<int, std::vector<int>> compatible_sub_tiles_map;
```

Comment:
We'll need a comment here noting what this is storing, what the key/value are, and its overall purpose. May also need a more descriptive name (compatible_sub_tiles_map gets across some of the idea, but not all).

Reply:
Done. I am unable to find a better name for this, I replaced it with `compatible_sib_tiles_for_tile_map` which sounds maybe even worse than before. Do you have any suggestions?